analysis_options.yaml

```diff
@@ -114,7 +115,8 @@ linter:
     - unnecessary_parenthesis
     - unnecessary_statements
     - unnecessary_this
+    - use_function_type_syntax_for_parameters
     - use_setters_to_change_properties
     - use_string_buffers
     - use_to_and_as_if_applicable
-#    - void_checks # under review (see #1068)
+    - void_checks # under review (see #1068)
```

Comment:
Perhaps it is not under review anymore :)

Reply:
Quite right.  Good catch!